### PR TITLE
"Gablebee", a blocklayer package for gandi DNS 

### PR DESCRIPTION
**Pull request by dubo-dubon-duponey**
Title says all.

I'm using this personally to manage my own DNS zones.

Happy to amend this PR to better fit the local style.

Let me know your thoughts.

Bonus: demo gif, so, it must be good, right? :p

gablebee

### CODE REVIEW

File: pkg/gandidns/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  ttl: uint32 | * 300
+
+  // The domain name to which this zone will be attached
+  domain: string
+
+  // By default ("replace"), the entire zone is being replaced by the provided records.
+  // In the "merge" mode, only the records that have the same "type" and "name" will be replaced,
+  // and the others will be kept.
+  // Note that NS records are ALWAYS left untouched and you can't specify them right now
+  mode: =~ "^(replace|merge)$" | * "replace"
```
Comment (shykes) [inline pkg/gandidns/README.md]: Nit: you can do this without a regexp:

```
mode: *"replace" | "merge"
```

File: pkg/gandidns/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  provider: {
+    settings: {
+      // Your Gandi API key - see Gandi documentation
```
Comment (shykes) [inline pkg/gandidns/README.md]: Nit: bonus points for an http link to the Gandi docs :)

**Review by shykes — APPROVED** [review]
LGTM, thanks! I added a few optional nitpicks :)

File: pkg/gandidns/README.md (hunk 1 above)
Comment (dubo-dubon-duponey) [reply pkg/gandidns/README.md]: Fixed

File: pkg/gandidns/README.md (hunk 2 above)
Comment (dubo-dubon-duponey) [reply pkg/gandidns/README.md]: And fixed!

**Comment by dubo-dubon-duponey** [comment]
@shykes want we to amend / squash, or multiple commits is fine?

**Comment by shykes** [comment]
@dubo-dubon-duponey multiple commits is fine.

**Pull request merged**